packages/integration-sdk-core/src/types/step.ts

```diff
@@ -240,3 +240,10 @@ export type StepMetadata = StepGraphObjectMetadataProperties & {
    */
   ingestionSourceId?: string;
 };
+
+export type StepWrapperFunction<
+  TStepExecutionContext extends StepExecutionContext,
+> = (
+  step: Step<TStepExecutionContext>,
```

Comment:
Do you foresee ever needing anything beyond `step` in the future? Sometimes making the first argument an object allows simple extension in the future, in case we need to add something (e.g., a logger) in the future.

```suggestion
  context: { 
    step: Step<TStepExecutionContext>,  
    // logger: IntegrationLogger,
    // context: TStepExecutionContext,
  }
```

Reply:
It can't hurt, that's for sure. Will update accordingly